s3backgrounddelete/s3backgrounddelete/object_recovery_msgbus.py

```diff
@@ -245,3 +245,23 @@ def purge(self):
             self._logger.error("Exception:{}".format(exception))
             self.__isproducersetupcomplete = False
             return False
+            
+    def get_count(self):
+        """Get count of unread messages."""
+        try:
+            consumer_group = self._config.get_msgbus_consumer_group()
+            if not self.__isproducersetupcomplete:
+                self.__setup_producer()
+                if not self.__isproducersetupcomplete:
+                    self._logger.info("producer connection issues")
+                    return False
+            if not self.__msgbuslib:
+                ret = self.__loadmsgbuslibrary()
+                if not ret:
+                    self.logger.debug("failed to load msgbus library")
+                    return False
```

Comment:
This function is returning False at places, integer at other place and nothing when an exception occurs. This is not how it should work.
Rewrite the function so that the function returns 0 on failure and count on success.
@vimalraghwani20 what are your thoughts?

Reply:
> This function is returning False at places, integer at other place and nothing when an exception occurs. This is not how it should work.
> Rewrite the function so that the function returns 0 on failure and count on success.
> @vimalraghwani20 what are your thoughts?

agree. this function should return only integer

Reply:
done